src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -294,6 +314,10 @@ var ReactElementValidator = {
 
     validatePropTypes(element);
 
+    if (__DEV__) {
```

Comment:
note: c508d90 also removes logic to reset `ReactDebugCurrentFrame.element` (since fiber/debug tracking has been decoupled from core implementation into renderers 3b5e3b5a23069dd03feaa6979bccb64de4d76d09)  ~ L255:

```diff
   if (__DEV__) {
-    ReactDebugCurrentFrame.element = null;
     freezeDefaultProps(element);
   }
```

i lost commit somehow in rebase, not too important but documenting for posterity